Makefile

```diff
@@ -0,0 +1,47 @@
+TEST_ARGS ?=
+TEST_ENVS ?=
+TEST_REBUILD ?= yes
+
+# target: help - Display callable targets.
+.PHONY: help
+help:
+	@echo "Reference card for usual actions in development environment."
+	@echo "Here are available targets:"
+	@egrep -o "^# target: (.+)" [Mm]akefile  | sed 's/# target: / * /'
+
+# target: install - Install workalendar in your current env
+.PHONY: install
+install:
+	pip install -e ./
```

Comment:
Usually we make sure to create the venv if the user is not already in a venv. See: 

- https://github.com/Kinto/kinto/blob/master/Makefile#L5-L6
- https://github.com/Kinto/kinto/blob/master/Makefile#L54-L56
- https://github.com/Kinto/kinto/blob/master/Makefile#L38